src/software/ai/navigator/path_planner/global_path_planner.cpp

```diff
@@ -0,0 +1,59 @@
+#include "global_path_planner.h"
```

Comment:
this should be the full path starting with "software/

Reply:
done